installer/roles/kubernetes/tasks/main.yml

```diff
@@ -185,14 +185,48 @@
   include_tasks: ssl_cert_gen.yml
   when: "rabbitmq_use_ssl|default(False)|bool"
 
-- name: Get Kubernetes API version
+- name: Get Kubernetes Config
   command: |
-    {{ kubectl_or_oc }} version -o json
+    {{ kubectl_or_oc }} config view -o json
+  register: kube_config_cmd
+  no_log: yes
+
+- name: Convert kube config to dictionary
+  set_fact:
+    kube_config: "{{ kube_config_cmd.stdout | from_json }}"
+  no_log: yes
+
+- name: Extract current context from kube config
+  set_fact:
+    current_kube_context: "{{ kube_config['current-context'] }}"
+
+- name: Find cluster for current context
+  set_fact:
+    kube_cluster: |
```

Comment:
Does anyone have a better way of doing this that doesn't require installing jq or jmespath?